### PR TITLE
fix(Tooltip): Fix cannot read property getBoundingClientRect of null.

### PR DESCRIPTION
**Pull request by stefhatcher**
to: @milesj @stefhatcher @hayes @alecklandgraf

## Description

Fixes a JS error where body content is not fully loaded (null) and `getBoundingClientRect` is called. Not sure how, but seeing this error.

## Motivation and Context

Fixing errors.

## Testing

Works still in the styleguide.

## Checklist

- My code follows the style guide of this project.
- I have updated or added documentation accordingly.
- I have read the CONTRIBUTING document.

### CODE REVIEW

File: packages/core/src/components/Tooltip/index.tsx [hunk 1]
```diff
@@ -58,7 +67,7 @@ export class Tooltip extends React.Component<Props & WithStylesProps, State> {
     labelID: uuid(),
     open: false,
     tooltipHeight: 0,
-    targetRect: document.body.getBoundingClientRect(),
+    targetRect: document.body ? document.body.getBoundingClientRect() : defaultTargetRect,
```
Comment (milesj) [inline packages/core/src/components/Tooltip/index.tsx]: I wonder if moving this to cDM would solve the problem.

Comment (stefhatcher) [reply packages/core/src/components/Tooltip/index.tsx]: moved it!

**Comment by stefhatcher** [comment]
updated @milesj

File: packages/core/src/components/Tooltip/index.tsx [hunk 2]
```diff
@@ -86,6 +86,13 @@ export class Tooltip extends React.Component<Props & WithStylesProps, State> {
     return null;
   }
 
+  componentDidMount() {
+    // eslint-disable-next-line
```
Comment (milesj) [inline packages/core/src/components/Tooltip/index.tsx]: Add the name of the rule this is disabling.

Comment (stefhatcher) [reply packages/core/src/components/Tooltip/index.tsx]: updated!

**Comment by stefhatcher** [comment]
updated @milesj

**Pull request merged**